stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.59.0
-appVersion: 1.7.6
+version: 1.60.0
+appVersion: 1.7.7
```

Comment:
@nreisbeck you also need to update here: https://github.com/helm/charts/blob/b6d71590913454cc9e4c0c814cbee0ad8faebb61/stable/traefik/README.md and here https://github.com/helm/charts/blob/b6d71590913454cc9e4c0c814cbee0ad8faebb61/stable/traefik/values.yaml#L3

Reply:
Whoops! Fixed both issues now.